news/2017-07-xx-Community-Mentorship-Program-Fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: no idea what to do with these fields
+date: 2017-07-xx xx:xx:xx +0000
```

Comment:
change to day 17 whatever time (24hrs) it is when you get around to changing this in UTC